### PR TITLE
Setup documentation deps

### PR DESCRIPTION
**Pull request by gnomex**
Bootstrapping documentation tools for #1

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -12,8 +15,17 @@ Metrics/BlockLength:
...
+Metrics/ClassLength:
   Enabled: false
+
+Metrics/ModuleLength:
+  Enabled: false
+
+Metrics/BlockLength:
+  Exclude:
+    - 'spec/**/*.rb'
+
```
Comment (MatheusRich) [inline .rubocop.yml:30]: I think this cops are unnecessary
```suggestion
```

File: .rubocop.yml [hunk 2]
```diff
@@ -3,6 +3,9 @@ require:
 
 AllCops:
   NewCops: enable
+  Include:
+    - Guardfile
+    - Rakefile
```
Comment (MatheusRich) [inline .rubocop.yml:8]: Is this really needed?

**Review by MatheusRich — CHANGES_REQUESTED** [review]

File: .rubocop.yml (hunk 1 above)
Comment (MatheusRich) [inline .rubocop.yml:30]: I think this cops are unnecessary

File: .rubocop.yml (hunk 2 above)
Comment (gnomex) [reply .rubocop.yml:8]: they are not necessary, only if you want to customize them. Removing.

File: .rubocop.yml (hunk 1 above)
Comment (gnomex) [reply .rubocop.yml:30]: ok, removing

**Comment by gnomex** [comment]
@MatheusRich with the given configuration we have two offenses:

```
➜  benchable git:(master) ✗ bundle exec rubocop
Inspecting 11 files
.....CC....

Offenses:

lib/benchable.rb:6:1: C: Style/Documentation: Missing top-level module documentation comment.
module Benchable
^^^^^^
lib/benchable/benchmark.rb:8:3: C: Style/Documentation: Missing top-level class documentation comment.
  class Benchmark
  ^^^^^
```

Its possible to disable the documentation cop, what do you think about?

**Comment by MatheusRich** [comment]
@gnomex this offenses are expected, since there's no docs about this module/class.

You can provide a simple description for both of them and the cops will pass

**Comment by MatheusRich** [comment]
Thanks for your contribution! Merged!

**Pull request merged**